src/en/getting-started-jaas.md

```diff
@@ -0,0 +1,127 @@
+Title: Getting started with Juju as a Service
+
+# Getting started with Juju as a Service
+
+Juju as a Service (JAAS) is the fastest and easiest way to quickly get started
+modeling and deploying your cloud-based applications. When you use JAAS,
+Canonical manages the Juju infrastructure. This frees you to concentrate on
+your software and solutions. With JAAS you can deploy, configure, and operate
+your applications on AWS, GCE, and Azure.
+
+## Create an Ubuntu single sign on account
+
+JAAS uses [Ubuntu single sign on][ubuntuSSO] (SSO) for identity management. If you don’t
+already have an Ubuntu SSO account, create one. You will provide a username
+when registering which is the username that will be used by JAAS.
+
+## Obtain cloud credentials
+
+In order for JAAS to deploy your workload on the cloud of your choice, you must
+enter credentials for each cloud you would like to use.
+
+We recommend that you use each public cloud's identity and access management
+(IAM) tool to generate a new set of credentials exclusively for use with JAAS.
+See [Cloud credentials][credentials] for more information. 
+
+## Use JAAS from the web GUI
+
+It is easiest to do the initial steps using the web. Open the [JAAS login page][jaaslogin]
+to begin.
+
+Log in to JAAS using your Ubuntu SSO account. This will open a second tab in
+your web browser to log you in. If the tab does not open, check and disable any
+pop-up blockers that may be running and try again.
+
+Press the green + button on the page to design and deploy your model. Press
```

Comment:
could probably do with some more steps and explanation - this may well be first time using Juju.